horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/services/TestService.java

```diff
@@ -260,8 +267,15 @@ List<ExportedLabelValues> listLabelValues(
    void importTest(ObjectNode test);
 
    class TestListing {
+      public TestListing(){}
+
+      @JsonProperty(required = true)
       @Schema(description = "Array of Test Summaries")
       public List<TestSummary> tests;
+
+      @JsonProperty(required = true)
+      @Schema(description = "total number of tests")
```

Comment:
Is this the total number of `Tests` in the `TestListing` or the total number of `Tests` available server side?

Reply:
The total Tests available (depending on folder or roles) on the server side.

Reply:
@whitingjr please can you update the description so that it is clear

Reply:
yes good idea.